app/views/tasks/show.html.erb

```diff
@@ -10,7 +11,25 @@
       </div>
     </div>
     <h5 class='text-md mt-2'><%= "Description: #{@task.body}" %></h5>
-    <h5 class='text-md my-2'><%= "Status: #{@task.status ? 'Completed' : 'In Progress'}" %></h5>
-    <%= link_to '<- Back', root_url, { class: 'hover:underline' } %>
+    <h5 class='text-md mt-2'><%= "Status: #{@task.status ? 'Completed' : 'In Progress'}" %></h5>
```

Comment:
Improvement (You could work in a separate PR): 

This would be better to be done by using enum data type. Change column `status` to be integer and define `enum` in model.

See further more:
https://api.rubyonrails.org/classes/ActiveRecord/Enum.html
https://blog.saeloun.com/2022/01/05/how-to-use-enums-in-rails.html